jb/itch5/order_book_cache_aware.cpp

```diff
@@ -0,0 +1,151 @@
+#include "jb/itch5/order_book_cache_aware.hpp"
+
+// definition of tick_off_
+jb::itch5::tick_t jb::itch5::order_book_cache_aware::tick_off_ = 5000;
```

Comment:
I am even more inclined to use a configuration object after reading more of the code.